vote_text_drafts/rpip-30.md

```diff
@@ -0,0 +1,57 @@
+> Vote options:
+> - For
+> - Against
+> - Abstain
+
+# RPL Staking Rework (RPIP-30)
+
+This vote aims to align RPL reward spend with rETH capacity created.
+
+### Summary
+- RPL rewards will scale differently, with a focus on aligning RPL rewards with supplying rETH
+- It will be possible to withdraw down to 15% borrowed ETH
+- We will ramp to both of the above states
```

Comment:
```suggestion
- We will transition to the new states over six reward periods
```
Numbers are right, right? It was 6 periods?

Reply:
It's complicated. Over 6 periods except the 2-step withdrawal thing which needs a smart contract upgrade. Will add words.